### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by Courtny** [microsoft/atlas-design#163] [Component]
Pagination splits content into pages.

**Comment by wibjorn** [comment]
Wanted to point to #66 as well.

**Comment by wibjorn** [comment]
@Courtny  there's some discussion over when the right time to paginate is. Do we have an principals like: "If you are showing dynamic content that could be over 20 items, we should paginate"? I'm not suggesting that per se, but it would be nice to get an idea of when pagination is suggested.
